### PR TITLE
Implement WiX based MSI setup

### PR DESCRIPTION
**Pull request by alexhass**
As discussed in https://github.com/AdoptOpenJDK/openjdk-build/issues/459 I'm sharing the installer scripts here.

This supports parallel installs for several versions and builds optional multilingual installers (*.MUI.msi).

I will try to attach latest MSI files I have build for testers. Not sure if I can attach 1 GB files to a case.

### CODE REVIEW

**Comment by alexhass** [comment]
Upload seems to be limited to 10MB.

File: wix/readme.TXT [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Requirements for build environment:
+===================================
+
+1. Windows Installer XML (WiX) toolset, 3.6 or later, http://wixtoolset.org/releases/
+2. Windows SDK for building multi-lingual setups (optional).
+3. Digital signature service if the MSI should be signed (optional).
+
+
+How to upgrade to a new Skype Version:
```
Comment (karianna) [inline wix/readme.TXT]: I think Skype is a typo here

File: wix/Resources/license.rtf [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Version 2, June 1991 \par
+\par
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.  \par
+59 Temple Place - Suite 330, Boston, MA  02111-1307, USA\par
+\par
+Everyone is permitted to copy and distribute verbatim copies\par
+of this license document, but changing it is not allowed.\par
+\par
+Preamble\par
+The licenses for most software are designed to take away your freedom to share and change it. By contrast, the GNU General Public License is intended to guarantee your freedom to share and change free software--to make sure the software is free for all its users. This General Public License applies to most of the Free Software Foundation's software and to any other program whose authors commit to using it. (Some other Free Software Foundation software is covered by the GNU Library General Public License instead.) You can apply it to your programs, too. \par
```
Comment (karianna) [inline wix/Resources/license.rtf]: I'm not sure this is the GPLv2+CPE license.  It looks like it's just the GPLv2?  Also need to check with the Eclipse OpenJ9 folks as to what license would be deployed for them.

File: wix/Build.OpenJDK8.withHotspot.cmd [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    REM CALL BuildSetupTranslationTransform.cmd ko-kr 1042
+    REM CALL BuildSetupTranslationTransform.cmd ru-ru 1049
+    CALL BuildSetupTranslationTransform.cmd zh-cn 2052
+    CALL BuildSetupTranslationTransform.cmd zh-tw 1028
+
+    REM Add all supported languages to MSI Package attribute
+    CSCRIPT "%ProgramFiles(x86)%\Windows Kits\%WIN_SDK_VERSION%\bin\x64\WiLangId.vbs" ReleaseDir\!OUTPUT_BASE_FILENAME!.MUI.msi Package !LANGIDS!
+
+    REM SIGN the MSIs with digital signature.
+    REM Dual-Signing with SHA-1/SHA-256 requires Win 8.1 SDK or later.
```
Comment (karianna) [inline wix/Build.OpenJDK8.withHotspot.cmd]: I think we have the correct certs etc in this repo - are you able to attempt signing?

File: wix/readme.TXT (hunk 1 above)
Comment (alexhass) [reply wix/readme.TXT]: You are right... the file has been copied over and than changed. This is a leftover.

File: wix/Resources/license.rtf (hunk 2 above)
Comment (alexhass) [reply wix/Resources/license.rtf]: Oh, thats correct. I started with the license included in the hotspot package and missed to review the other package about this. Just thought it is the same.

File: wix/Build.OpenJDK8.withHotspot.cmd (hunk 3 above)
Comment (alexhass) [reply wix/Build.OpenJDK8.withHotspot.cmd]: I do not have access to the private keys. Signing the MSI would be great, but has not tested yet. I only added the commands for it.

**Comment by karianna** [comment]
OK, I'm going to merge this a raise a couple of issues to cover the license, signing and integrating this into our pipelines.

**Pull request merged**